src/main/app/claims/models/claim.ts

```diff
@@ -72,13 +72,21 @@ export class Claim implements Serializable<Claim> {
     return this.settlement.getDefendantOffer()
   }
 
-  get totalAmount (): number {
+  get totalAmountTillToday (): number {
```

Comment:
Please add unit tests to cover `totalAmountTillToday` and `totalAmountTillDateOfIssue` methods. No tests there at this point.

Reply:
Yes I will do, strange there wasn't an existing test for the refactored and renamed function totalAmount